job-dsls/src/main/resources/job-scripts/prod_bamoe_kogito_stage_cpaas_artifacts.jenkinsfile

```diff
@@ -86,8 +88,8 @@ def createOfflinerZip() {
 
     remoteExec("mkdir -p ${offlinerFolderPath}")
```

Comment:
You will need the full path here (RCM_GUEST_FOLDER)  as it is going to create the folder.

If you need a relative path because of rsync (due to the way spmm-utils host is configured), maybe you can define this variable inside the `util.withKerberos('rhba-prod-keytab') {` block.